### DISCUSSION TITLE
Sobre nota: Los grupos que tengan 6 o 7 miembros deberán trabajar en dos subgrupos de mínimo 3 personas cada uno

### DISCUSSION THREAD

**Discussion by D-M-Gonzalez** [dds-jv/foro#433] [TPA - 2024]
Que tal

Me levanta muchas dudas esta solicitud. ¿Cómo es la dinámica de esto? ¿No nos podemos ayudar entre los grupos? ¿No se pueden cruzar tareas? ¿Qué pasa si un subgrupo no logra cumplir los objetivos? Lamentablemente este viernes no tenemos clase como para que se aclaren las dudas, pero estaría bueno que expliquen que dinámica tienen pensada.

**Comment by flbulgarelli** [comment]
¡Hola!

La intención es repartir las tareas de los grupos grandes de forma que sean de complejidad más o menos equitativa. El reparto de tareas fue hecho procurando que las mismas estén relacionadas entre si y que sean relativamente faciles de distinguir. 

La intención es que el diseño de toda la entrega lo realicen  en equipo, como siempre, pero que la implementación tenga personas claramente asignadas, así les resulta más sencillo al equipo de trabajo el reparto, pero también al equipo docente su evaluación y la evaluación individual.

Buscamos además que de esta forma todas las personas puedan participar.

Teniendo en cuenta todo esto, la nota de cada subgrupo es independiente, y puede perfectamente aprobar un subgrupo y el otro no. Por otro lado, si se evaluará como grupo completo que el diseño de todas las partes sea compatible entre si, con lo que no se aceptarán entregas en dónde el diseño de un subgrupo no dialoga adecuadamente con el del otro. 

Sumado a esto, siguen aplicando las consideraciones usuales de una nota individual: cada persona puede tener una nota diferenciada, independientemente de como le vaya al grupo en general o a su subgrupo en particular.

Espero con esto haberte respondiendo todas las inquietudes.

¡Saludos!
